containers/podman-desktop-extension-bootc
# docs: use gifs, link to videos

docs: use gifs, link to videos

### What does this PR do?

Because we cannot directly link to raw files hosted on github, instead
use gifs and link to the video.

### Screenshot / video of UI



See README.md

### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop-extension-bootc/issues/113

### How to test this PR?

See README.md, click on an image and it'll take you to the video.



Signed-off-by: Charlie Drage <charlie@charliedrage.com>
